apps/pano/app/features/comment/EditCommentForm.tsx

```diff
@@ -22,14 +28,26 @@ export const EditCommentForm: FC<EditCommentProps> = ({
     commentContent: editedComment,
   };
 
+  const validationResult = commentSchema.safeParse({ content: editedComment });
+  const errorMessage = validationResult.success
+    ? null
+    : validationResult.error?.errors[0].message;
+
   useEffect(() => {
     if (fetcher.type === "done") {
       setEditOpen(false);
     }
   }, [fetcher.type, setEditOpen]);
 
   return (
-    <fetcher.Form method="post" action="/commentEdit">
+    <fetcher.Form
+      method="post"
+      action="/commentEdit"
+      onSubmit={(event) => {
```

Comment:
do we need to add onSubmit here?

Reply:
i noticed that if a user removes the "disabled" attribute from the inspect menu, the form becomes submit-able. however, using onSubmit may not be very effective.

Reply:
I noticed that if user removes the "disabled" attribute from the inspect menu, the form becomes submit-able. if I'm not doing this correctly, I can make changes.

Reply:
I believe this shouldn't be a problem as long as we validating input on the server side as well,  but i'm not sure which option is better tbh.

Reply:
Do you have any input on this? @usirin